chat-widget/src/components/livechatwidget/common/endChat.ts

```diff
@@ -18,20 +18,40 @@ import { defaultWebChatContainerStatefulProps } from "../../webchatcontainerstat
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const prepareEndChat = async (props: ILiveChatWidgetProps, chatSDK: any, state: ILiveChatWidgetContext, dispatch: Dispatch<ILiveChatWidgetAction>, setAdapter: any, setWebChatStyles: any, adapter: any, uwid: string) => {
     try {
+        // Use Case: If call is ongoing, end the call by simulating end call button click
+        const voiceVideoCallingSdk = await chatSDK.getVoiceVideoCalling();
```

Comment:
There's no type for this SDK here. How can we make sure voiceVideoCallingSdk.isInACall() or voiceVideoCallingSdk.stopCall() is a valid call?

Reply:
Maybe move the try catch outside the "isInACall" call?

Reply:
Moved try catch